docs/Remote.rst

```diff
@@ -79,6 +79,7 @@ from other (non-C++) languages, including:
 - `dfhack-client-qt <https://github.com/cvuchener/dfhack-client-qt>`_ for C++ with Qt
 - `dfhack-client-python <https://github.com/McArcady/dfhack-client-python>`_ for Python (adapted from :forums:`"Blendwarf" <178089>`)
 - `dfhack-client-java <https://github.com/McArcady/dfhack-client-java>`_ for Java
+- `dfhack-remote <https://docs.rs/dfhack-remote/latest/dfhack_remote/index.html>`_ for Rust
```

Comment:
I think you can get around the link name conflict just by using a double underscore at the end instead of a single underscore. You might need to do it for the other dfhack-remote link too.

Reply:
It fixed it, thanks!